### PR TITLE
[14.0] [mig] base_export_manager

### PR DESCRIPTION
**Pull request by schhatbar-initos**

### CODE REVIEW

File: base_export_manager/views/res_groups.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="res_groups_view_form" model="ir.ui.view">
```
Comment (dsolanki-initos) [inline base_export_manager/views/res_groups.xml]: You should use original_id here as per the guideline.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views

Comment (dsolanki-initos) [inline base_export_manager/views/res_groups.xml]: You should use original_id here as per the guideline.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views

File: base_export_manager/views/ir_model_access.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="ir_model_access_view_tree" model="ir.ui.view">
```
Comment (dsolanki-initos) [inline base_export_manager/views/ir_model_access.xml]: Original Id should be used

File: base_export_manager/views/ir_model.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="ir_model_view_form" model="ir.ui.view">
```
Comment (dsolanki-initos) [inline base_export_manager/views/ir_model.xml]: same here

**Comment by schhatbar-initos** [comment]
@Tardo Can you please review this PR?

**Comment by schhatbar-initos** [comment]
@LoisRForgeFlow Can you please review this PR?
Thanks in Advance!

**Comment by schhatbar-initos** [comment]
@rousseldenis Can you please review this PR?
Thanks in Advance!

**Comment by schhatbar-initos** [comment]
@all can someone please review this PR?

**Comment by fshah-initos** [comment]
Can somone please review the PR?

**Comment by fshah-initos** [comment]
It would be nice if somone can add reviews so that PR can be merged ASAP.

**Comment by fshah-initos** [comment]
It would be great if we can get review here and merge the PR soon.

**Comment by LoisRForgeFlow** [comment]
@fshah-initos It is missing reviews, you can do a review yourself to help the module get merged.

**Review by fshah-initos — APPROVED** [review]
LGTM.
Functional Tests and code review.

**Comment by fshah-initos** [comment]
It would be nice if we could merge the PR.

**Comment by fshah-initos** [comment]
It would be nice if we could merge the PR.

**Comment by LoisRForgeFlow** [comment]
@fshah-initos I'm sorry but this is still missing one more review, and the commit history is not clean:
image

there should be no merge commits in the history.

**Comment by simahawk** [comment]
already migrated here https://github.com/OCA/server-ux/pull/227

**Pull request closed without merging**